src/RealtimeChannel.ts

```diff
@@ -57,22 +57,24 @@ export type RealtimePostgresChangesPayload<T extends { [key: string]: any }> =
   | RealtimePostgresUpdatePayload<T>
   | RealtimePostgresDeletePayload<T>
 
-export type RealtimePostgresChangesFilter<T extends `${REALTIME_POSTGRES_CHANGES_LISTEN_EVENT}`> = {
+export type RealtimePostgresChangesFilter<
+  T extends `${REALTIME_POSTGRES_CHANGES_LISTEN_EVENT}`
+> = {
   /**
-  * The type of database change to listen to.
-  */
+   * The type of database change to listen to.
```

Comment:
I believe these changes came from running `npm run format`, please let me know if you want them backed out.